packages/material-ui-lab/package.json

```diff
@@ -38,8 +38,10 @@
     "react-dom": "^16.8.0"
   },
   "dependencies": {
+    "@babel/register": "^7.4.4",
     "@babel/runtime": "^7.2.0",
     "@material-ui/utils": "^4.0.0-alpha.4",
+    "babel": "^6.23.0",
```

Comment:
?

Reply:
what version of babel should there be? I'm not sure how this happened.

Reply:
> what version of babel should there be? 

None? 🙂

Reply:
Haha ok, I'll make the change now

Reply:
done! thanks